configure.py

```diff
@@ -710,4 +710,8 @@ def has_re2c():
     if options.verbose:
         rebuild_args.append('-v')
 
-    subprocess.check_call(rebuild_args)
```

Comment:
Not sure that it would be worth updating configure.py to support ninja as a library, the CMake integration seems to be the favored way forward.